packages/frontend/src/components/wallet-migration/modals/CleanKeysModal/CleanKeysModal.jsx

```diff
@@ -33,9 +33,11 @@ async function getAccountDetails({ accountId, publicKeyBlacklist, wallet }) {
         }, {});
 
     const allAccessKeys = await wallet.getAccessKeys(accountId);
+    const currentAccount = await wallet.getLocalAccessKey(accountId, allAccessKeys);
```

Comment:
Can we use `wallet.getPublicKey(accountId)` here instead? It's more consistent with how we're getting the key elsewhere and is equivalent in this case.

The usage here is a little circuitous since `getLocalAccessKey` only returns a value if it's in `allAccessKeys`, but we only want to get the key to filter it out of `allAccessKeys`.